packages/nimble-components/src/dialog/tests/dialog.stories.ts

```diff
@@ -49,9 +56,33 @@ const content = {
     [ExampleContentType.longContent]: longContent
 } as const;
 
+const sizeDescription = `
+Size of a nimble dialog.
+
+See the Sizing section below for information on controlling the size of the dialog.
+`;
+
+const widths = {
+    [DialogSizeOptions.smallGrowable]: dialogSmallWidth.getValueFor(
+        document.body
+    ),
+    [DialogSizeOptions.largeFixed]: dialogLargeWidth.getValueFor(document.body)
+} as const;
+
+const heights = {
+    [DialogSizeOptions.smallGrowable]: 'fit-content',
+    [DialogSizeOptions.largeFixed]: dialogLargeHeight.getValueFor(document.body)
```

Comment:
We shouldn't get the calculated value for all these, we should just leverage the design token css custom property. Browser should do the work for calculating it. i.e. `var(${dialogLargeHeight.cssCustomProperty})`